content/en/news/2019/announcing-1.1.17/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Announcing Istio 1.1.17
+description: Istio 1.1.17 patch release.
+publishdate: 2019-10-18
+attribution: The Istio Team
+release: 1.1.17
+---
```

Comment:
I know I just had you remove an alias in the other file, so sorry for making you add one here.

For the sake of having simpler links to release notes from istio/istio, all our release announcements now need a default alias:

```yaml
aliases:
    - /news/announcing-1.1.7
```
Thanks.

Reply:
done